scripts/template/src/patternfly/{folder}/{name}/examples/{name}.md

```diff
@@ -1,8 +1,7 @@
 ---
-title: '{{nameCapitalized}}'
-section: beta
+title: '{name}'
 beta: true
-cssPrefix: pf-c-{{nameCapitalized}}
+cssPrefix: {nameBEM}
 ---
 
 ## Examples
```

Comment:
In the example code below, the hbs in the "Basic" example is what gets created. So it creates

```
{{moduleHbOpen}}
  Basic example content
{{moduleHbClose}}
```

instead of

```
{{#> foo-bar}}
  Basic example content
{{/foo-bar}}
```

Reply:
Fixed.